pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -490,7 +491,10 @@ func buildInboundClustersFromServiceInstances(cb *ClusterBuilder, proxy *model.P
 		clustersToBuild[port] = append(clustersToBuild[port], instance)
 	}
 
-	bind := ""
+	bind := actualLocalHosts[0]
+	if cb.req.Push.Mesh.InboundTrafficPolicy.Mode == meshconfig.MeshConfig_InboundTrafficPolicy_PASSTHROUGH {
```

Comment:
won't cb.req.Push.Mesh.InboundTrafficPolicy usually be nil and crash here? Somehow we don't crash though :thinking:

Reply:
Oh we set a default everywhere.. that seems possibly problematic, can we make this null safe

Reply:
Done